tests/jballerina-unit-test/src/test/resources/test-src/bala/test_projects/bir_test_project/Ballerina.toml

```
@@ -0,0 +1,5 @@
+[package]
+org= "bir"
```

```suggestion
org = "bir"
```

Fix in the rest also.